rep-0150.rst

```diff
@@ -0,0 +1,212 @@
+REP: 150
+Title: ROS Melodic and Newer Metapackages
+Author: Chris Lalancette <clalancette@openrobotics.org>
```

Comment:
As this is mostly a copy-n-paste of an existing REP, my guess is that the original author should be mentioned (not sure how it works for REPs)?

Reply:
Yeah, I'm not entirely sure how it works.  Looking at REP-133, for instance, it has multiple authors, so I think I'll add multiple authors here.